#! /bin/sh
patch -p1 -l -f $* < $0
exit $?

From 45e07725c20d0d0e3b1a648975e277c52455029f Mon Sep 17 00:00:00 2001
From: Andre Heinecke <aheinecke@gnupg.org>
Date: Wed, 10 Apr 2024 11:02:17 +0200
Subject: [PATCH] Change check from Android to DBus check

This allows builds without DBus even on platforms
which theoretically can have a DBus, like Windows.
---
 src/readwritepart.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/readwritepart.cpp b/src/readwritepart.cpp
index 69873a1..8ae93c2 100644
--- a/src/readwritepart.cpp
+++ b/src/readwritepart.cpp
@@ -274,7 +274,7 @@ void ReadWritePartPrivate::slotUploadFinished(KJob *)
         }
         Q_EMIT q->canceled(error);
     } else {
-#ifndef Q_OS_ANDROID
+#ifdef QT_DBUS_LIB
         ::org::kde::KDirNotify::emitFilesAdded(m_url.adjusted(QUrl::RemoveFilename));
 #endif

--
2.44.0
